### PR TITLE
fixed code + added test

### PR DESCRIPTION
**Pull request by bthirion**
Closes # .

Changes proposed in this pull request:

-
-

### CODE REVIEW

**Comment by bthirion** [comment]
* fixes #2879 
* supersedes #2889

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks!
Maybe you can add a whatsnew entry?

**Review by jeromedockes — APPROVED** [review]
great!

File: doc/whats_new.rst [hunk 1]
```diff
@@ -68,6 +68,8 @@ Fixes
   index (See PR `#3078 <https://github.com/nilearn/nilearn/issues/3078>`_).
 - Convert reference in `nilearn/regions/region_extractor.py` to use footcite / footbibliography.
   (See issue `#2787 <https://github.com/nilearn/nilearn/issues/2787>`_ and PR `#3111 <https://github.com/nilearn/nilearn/pull/3111>`_).
+- Computation of Benjamini-Hocheberg threshold fixed in `nilearn/glm/thresholding.py` function (see issue ``#2879 <https://github.com/nilearn/nilearn/issues/2879>`_ and PR `#3137 <https://github.com/nilearn/nilearn/pull/3137>`_`)
```
Comment (NicolasGensollen) [inline doc/whats_new.rst:71]:
```suggestion
- Computation of Benjamini-Hocheberg threshold fixed in `nilearn/glm/thresholding.py` function (see issue `#2879 <https://github.com/nilearn/nilearn/issues/2879>`_ and PR `#3137 <https://github.com/nilearn/nilearn/pull/3137>`_)
```

**Comment by NicolasGensollen** [comment]
Thanks, merging...

**Pull request merged**